### DISCUSSION TITLE
No message content in inline queries

### DISCUSSION THREAD

**Discussion by tizero** [python-telegram-bot/python-telegram-bot#2439] [Q&A]
When I run the [inlinebot](https://github.com/python-telegram-bot/python-telegram-bot/blob/master/examples/inlinebot.py) provided in the `examples` the text in `input_message_content` is not displayed in the inline query, which only shows the title.

I'm receiving this error:

`telegram.error.BadRequest: Can't parse inline query result: input message content is not specified`

**Comment by Bibo-Joshi** [comment]
Do you actually type a query? What's the full traceback?

**Reply by tizero** [reply]
I figured it out.

The error has gone by adding `if not query: return` in the callback. Moreover, what I was actually looking for was the `description` parameter in InlineQueryResultArticle to display text after the title of the result.

Thank you!

**Reply by harshil21** [reply]
Was just fixed by #2441.
